capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.upcase
+  end
```

Comment:
- Check the reason why `CapitaliizeDecorator` is not working this might be caused that a class or module referred to in code can't be found
<img width="1073" alt="Screen Shot 2023-03-07 at 21 29 11" src="https://user-images.githubusercontent.com/44745855/223532960-e2807bcd-b1b5-430e-896b-fa959edef4a5.png">